bluesky/brittle_scans.py

```diff
@@ -0,0 +1,247 @@
+"""
```

Comment:
We have to rename thing module, we are being brats.

I suggest we go back to 'simple'